doc/pipeline_details.rst

```diff
@@ -47,8 +47,9 @@ Read the description for more information.
 |                       |                       | takes to receive a response the longer loads and stores     |
 |                       |                       | will stall.                                                 |
 +-----------------------+-----------------------+-------------------------------------------------------------+
-| Multiplication        | 2/3 (Fast Multiplier) | 2 for MUL, 3 for MULH.                                      |
+| Multiplication        | 1-3 (Fast Multiplier) | 2 for MUL, 3 for MULH (ASIC), 1 for MUL, 2 for MULH (FPGA). |
```

Comment:
Cycles for MUL/MULH depend purely on what parameters you've chosen. Up to the person using Ibex what they deem suitable for ASIC. So better to say it simply depends upon the MultiplierImplementation setting.

Reply:
I have updated the doc.

Reply:
Looks good, thanks